sycl/test/xocc_tests/disabled/test_device.cpp

```diff
@@ -0,0 +1,55 @@
+// Test that CUDA code and XOCC code can cooperate in a single SYCL application.
+
+#include <CL/sycl.hpp>
+
+template<typename S>
+struct FillBuffer {};
+
+template<typename Selector>
+void test_device(Selector s) {
+  // Creating buffer of 4 ints to be used inside the kernel code
+  cl::sycl::buffer<cl::sycl::cl_int, 1> Buffer(4);
+
+  // Creating SYCL queue
+  cl::sycl::queue Queue(s);
```

Comment:
```suggestion
  sycl::queue Queue{s};
```
for coherency with you other use of `{}`